SPECS/golang/golang.spec

```diff
@@ -14,8 +16,8 @@
 %define __find_requires %{nil}
 Summary:        Go
 Name:           golang
-Version:        1.22.7
-Release:        5%{?dist}
+Version:        1.24.7
```

Comment:
For `golang` bump, I think there are a lot more packages, which build with them. See this `golang` update for instance: #14619.

With that in mind, I'm very worried about this change, considering the large version jump. What is the exact requirement for it, since we have `msft-golang` on 1.24 already?